projects/plugins/boost/app/assets/src/js/pages/settings/sections/Score.svelte

```diff
@@ -95,15 +175,32 @@
 				<MobileIcon />
 				<div>{__( 'Mobile score', 'jetpack-boost' )}</div>
 			</div>
-			<ScoreBar bind:score={scores.mobile} active={siteIsOnline} {isLoading} />
+			<ScoreBar
+				prevScore={scores.previous?.mobile}
+				score={scores.current.mobile}
```

Comment:
I have removed `bind` here. We don't need two way data flow. Besides, `prevScore` can benifit from optional chaining, which is not possible with bind.